src/effects/__tests__/journal.effects.spec.ts

```diff
@@ -0,0 +1,56 @@
+
+import { TestBed } from '@angular/core/testing';
+
+import { JournalEffects } from '../journal.effects';
+import { Actions } from '@ngrx/effects';
+import { Observable } from 'rxjs';
+import { empty } from 'rxjs/observable/empty';
+import { JournalProvider } from '../../providers/journal/journal';
+import { JournalProviderMock } from '../../providers/journal/__mocks__/journal.mock';
+
+export class TestActions extends Actions {
+  constructor() {
+    super(empty());
+  }
+
+  set stream(source: Observable<any>) {
+    this.source = source;
+  }
+}
+
+export function getActions() {
+  return new TestActions();
+}
+
+describe('Journal Effects', () => {
+
+  // @ts-ignore
+
+  let effects: JournalEffects;
+
+  beforeEach(() => { 
+    TestBed.configureTestingModule({
+      imports: [
+        // any modules needed
+      ],
+      providers: [
+        JournalEffects,
+        {
+          provide: Actions,
+          useFactory: getActions
+        },
+        { provide: JournalProvider, useClass: JournalProviderMock }
+      ]
+    });
+
+    effects = TestBed.get(JournalEffects);
+  })
+
+  it('should create the journal effects', () => {
+    expect(effects).toBeTruthy();
+  });
+
+  it('should load journal data', () => {
+    // todo - scaffold the load journal test
```

Comment:
This needs an entirely different story with allocated story points because it is not that easy that it seems. I suggest we come back to this after we crack the effects unit testing riddle.